shop/src/main/java/ro/msg/learning/shop/service/CustomerService.java

```diff
@@ -0,0 +1,40 @@
+package ro.msg.learning.shop.service;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import ro.msg.learning.shop.domain.Customer;
+import ro.msg.learning.shop.dto.CustomerDto;
+import ro.msg.learning.shop.mapper.CustomerMapper;
+import ro.msg.learning.shop.repository.CustomerRepository;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+
+@RequiredArgsConstructor
+@Service
+public class CustomerService {
+
+    private final CustomerRepository customerRepository;
+
+    private final CustomerMapper customerMapper;
+
+    public CustomerDto createCustomer(Customer customer) {
```

Comment:
`CustomerService` still uses DTOs. For consistency, it should use only entities.